# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/webrtc/signaling/src/peerconnection/RTCStatsReport.h
# Commit: 8d554870dce8
# Full Hash: 8d554870dce84fbcfefd842c235efcaa5aae23db
# Author: Tom Tung <ttung@mozilla.com>
# Date: 2020-04-07 21:44:02
# Regressor Bug: 1586761
# File Overlap Count: 1
# Description:
#   Bug 1586761 - P5 - Use new methods in media r=tjr,baku
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D63906
# ==============================================================================

diff -r f185ed882616 -r 8d554870dce8 media/webrtc/signaling/src/peerconnection/RTCStatsReport.h
--- a/media/webrtc/signaling/src/peerconnection/RTCStatsReport.h	Wed Mar 25 14:14:37 2020 +0000
+++ b/media/webrtc/signaling/src/peerconnection/RTCStatsReport.h	Wed Mar 25 14:16:15 2020 +0000
@@ -35,6 +35,7 @@
   uint64_t mRandomTimelineSeed = 0;
   DOMHighResTimeStamp mStartWallClockRaw = (double)PR_Now() / PR_USEC_PER_MSEC;
   TimeStamp mStartMonotonic = TimeStamp::NowUnfuzzed();
+  bool mCrossOriginIsolated = false;
 };
 
 typedef MozPromise<UniquePtr<RTCStatsCollection>, nsresult, true>
